Rubberduck.Deployment/InnoSetup/Installer Build Script.iss

```diff
@@ -0,0 +1,691 @@
+#define AppVersion GetFileVersion(BuildDir + "Rubberduck.dll")
+#define AppPublisher "Rubberduck"
+#define AppURL "http://rubberduckvba.com"
+#define License SourcePath + "\License.rtf"
+#define OutputDirectory SourcePath + "Installers\"
+#define AddinProgId "Rubberduck.Extension"
+#define AddinCLSID "8D052AD8-BBD2-4C59-8DEC-F697CA1F8A66"
+
+; Output the defined constants to aid in verification
+#pragma message "Include: " + __INCLUDE__
+#pragma message "Config: " + Config
+#pragma message "SourcePath: " + SourcePath
+#pragma message "BuildDir: " + BuildDir
+#pragma message "AppName: " + AppName
+#pragma message "AddinDLL: " + AddinDLL
+#pragma message "DllFullPath: " + DllFullPath
+#pragma message "Tlb32bitFullPath: " + Tlb32bitFullPath
+#pragma message "Tlb64bitFullPath: " + Tlb64bitFullPath
+#pragma message "AppVersion: " + AppVersion
+#pragma message "AppPublisher: " + AppPublisher
+#pragma message "AppURL: " + AppURL
+#pragma message "License: " + License
+#pragma message "OutputDirectory: " + OutputDirectory 
+#pragma message "AddinProgId: " + AddinProgId
+#pragma message "AddinCLSID: " + AddInCLSID
+
+[Setup]
+; TODO this CLSID should match the one used by the current installer.
+AppId={{979AFF96-DD9E-4FC2-802D-9E0C36A60D09}
+AppName={#AppName}
+AppVersion={#AppVersion}
+AppPublisher={#AppPublisher}
+AppPublisherURL={#AppURL}
+AppSupportURL={#AppURL}
+AppUpdatesURL={#AppURL}
+DefaultDirName={code:GetDefaultDirName}
+DefaultGroupName=Rubberduck
+DisableProgramGroupPage=yes
+AllowNoIcons=yes
+LicenseFile={#License}
+OutputDir={#OutputDirectory}
+OutputBaseFilename=Rubberduck.Setup
+Compression=lzma
+SolidCompression=yes
+
+ArchitecturesAllowed=x86 x64
+ArchitecturesInstallIn64BitMode=x64
+
+SetupLogging=yes
+PrivilegesRequired=lowest
+
+; should be a 164 x 314 bitmap
+WizardImageFile=InstallerBitMap.bmp
+
+[Languages]
+Name: "English"; MessagesFile: "compiler:Default.isl"
+Name: "French"; MessagesFile: "compiler:Languages\French.isl"
+Name: "German"; MessagesFile: "compiler:Languages\German.isl"
+
+[Files]
+; Install the correct bitness binaries.
+; Source: "{#BuildDir}*"; DestDir: "{app}"; Flags: ignoreversion recursesubdirs replacesameversion; Permissions: users-readexec;
+Source: "{#BuildDir}*"; DestDir: "{app}"; Flags: ignoreversion recursesubdirs replacesameversion; Excludes: "Rubberduck.Deployment.*,Rubberduck.dll.xml,Rubberduck.x32.tlb.xml,{#AddinDLL},\NativeBinaries"; Check: CheckShouldInstallFiles
+Source: "{#BuildDir}{#AddinDLL}"; DestDir: "{app}"; Flags: ignoreversion replacesameversion; Check: CheckShouldInstallFiles
+
+[Registry]
+; DO NOT attempt to register VBE Add-In with this section. It doesn't work
+; Use [Code] section (RegisterAddIn procedure) to register the entries instead.
+#include <Rubberduck.reg.iss>
+
+[UninstallDelete]
+Type: filesandordirs; Name: "{localappdata}\{#AppName}"
+
+[CustomMessages]
+; TODO add additional languages here by adding include files in \Includes folder

```

Comment:
Please note that some internationalization help is needed. The installer itself already has French and German enabled but those custom messages needs to be translated. See [this file](https://github.com/rubberduck-vba/Rubberduck/pull/3836/files#diff-cc583202f7e5d0c1bd632f6290b7615b) for a sample of how it should be authored. I made it an include file hopefully to make it easier to localize.